config/base.yaml

```diff
@@ -1,31 +1,32 @@
-### Security Server Toolkit Configuration Settings
-
+api-key:
+- key: /path/to/ssh_private_key
```

Comment:
- Documentation mentions  ``XROAD_SYSTEM_ADMINISTRATOR``, IIRC ``XROAD_SECURITY_OFFICER`` is also required (token ops). Put the defaults into the configuration here, not just placeholders.
- Is there any scenario, where default URL would NOT give desired result, that this has to be configured at all?
- ``key`` preceded by ``api-key`` does not seem intuitive. Something along the lines of 'host-access-key' maybe?

Reply:
Probably the roles can be as default in the configuration file as these are really the same with each run. The url is probably the same aswell and I could change the name of the "key" parameter here, could be a bit misleading.

Reply:
It has been agreed, that placeholders will be used for values in the configuration files. The documentation of how the configuration and running of the toolkit will happen will be further addressed in SSTOOLKIT-6, when we create the User Guide for the Toolkit. I would not place any default values there also in case something will be changed for the SS API, for example some other role has to be used, then we will just need to update the user guide. For the same reason, I would still keep the "url" there, as some parts of that may change and if we would hard-code it to somewhere else, it would not be so simple to change (when for example the url will be changed to https://localhost:5000... or https://localhost:4000/api/v2...). Maybe we should also document about this url in the User Guide, that this is for just creating the api keys and there is no need to change it by users.

Reply:
Ok, I am fine with defaults on hold now, until SSTOOLKIT-6 -- however these values are very unlikely to change in most configurations within _years_ to come -- note that removal of the configuration was not suggested, just simplifying the configuration hoops for cross-referencing documentation by providing default values.

I myself have configuration copy at hand and it does not affect me very much, lets' see what feedback the first users will give. Note that this new configuration is such that placeholders do not allow straightforward replacement anymore, as ``SECURITY_SERVER_ROLE_NAME`` takes multiple values and is listed twice.